src/model/game/GameManager.ts

```diff
@@ -794,6 +794,12 @@ export default class GameManager {
             "https://thunderstore.io/c/gorebox/api/v1/package-listing-index/", EXCLUSIONS,
             [new StorePlatformMetadata(StorePlatform.STEAM, "2027330")], "GoreBox.png",
             GameSelectionDisplayMode.VISIBLE, GameInstanceType.GAME, PackageLoader.BEPINEX, [""]),
+        
+        new Game("TCG Card Shop Simulator", "TCG Card Shop Simulator", "TCG Card Shop Simulator",
+            "TCG Card Shop Simulator", ["Card Shop Simulator.exe"], "Card Shop Simulator_Data",
+            "thunderstore.io/c/tcg-card-shop-simulator/api/v1/package/", "https://raw.githubusercontent.com/ebkr/r2modmanPlus/master/modExclusions.md",
+            [new StorePlatformMetadata(StorePlatform.STEAM, "3070070"), new StorePlatformMetadata(StorePlatform.OTHER)], "TCG Card Shop Simulator.png",
```

Comment:
- What is the other platform the game is available on?
- The convention is for the `gameImage` parameter to not include spaces

Reply:
First one was likely an oversight on my end Inhad that there in the case that other platforms would pop up similar to Inscryption. Also understood will change the image name.

Reply:
Should be resolved now [Keeping the Other]